app/code/Magento/Ui/etc/db_schema.xml

```diff
@@ -18,8 +18,10 @@
                 comment="Mark current bookmark per user and identifier"/>
         <column xsi:type="varchar" name="title" nullable="true" length="255" comment="Bookmark title"/>
         <column xsi:type="longtext" name="config" nullable="true" comment="Bookmark config"/>
-        <column xsi:type="datetime" name="created_at" on_update="false" nullable="false" comment="Bookmark created at"/>
-        <column xsi:type="datetime" name="updated_at" on_update="false" nullable="false" comment="Bookmark updated at"/>
+        <column xsi:type="timestamp" name="created_at" on_update="false" nullable="false" default="CURRENT_TIMESTAMP"
```

Comment:
Unfortunately, we are not allowed to change column type according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#db-schema)

Reply:
@sidolov ok I will check if it is possible to fix the issue without changing the column type